docker/kafka/Dockerfile

```diff
@@ -37,4 +37,4 @@ COPY --from=base /kafka/storage/api/build/libs/kafka-storage-api-${KAFKA_VERSION
 ENV KAFKA_HOME /kafka
 WORKDIR ${KAFKA_HOME}
 ENV PATH $PATH:${KAFKA_HOME}/bin
-CMD [ "kafka-server-start.sh" ]
\ No newline at end of file
+CMD [ "kafka-server-start.sh" ]
```

Comment:
nit: just wondering if this is intentional or sometimes github get's confused with new lines at EOF.

Reply:
Not intentional but new lines at the end are required in most of our projects and when touching file having IDE configured to add them this happens sometimes :) I would prefer to keep it if you don't mind, despite the fact that it's unrelated to PR.

Reply:
Hm, weird.